TypeSystem/Schema/smoke_test_struct.h

```diff
@@ -52,20 +46,7 @@ CURRENT_STRUCT(FullTest) {
   CURRENT_FIELD(q, (Variant<A, B, C, Empty>));
   CURRENT_CONSTRUCTOR(FullTest)(Variant<A, B, C, Empty> && q) : q(std::move(q)) {}
 };
-#else
-CURRENT_STRUCT(FullTest) {
-  CURRENT_FIELD(primitives, Primitives);
-  CURRENT_FIELD(v1, std::vector<std::string>);
-  CURRENT_FIELD(v2, std::vector<Primitives>);
-  typedef std::pair<std::string, Primitives> t_p;
-  CURRENT_FIELD(p, t_p);
-  CURRENT_FIELD(o, Optional<Primitives>);
-  typedef Variant<A, B, C, Empty> t_q;
-  CURRENT_FIELD(q, t_q);
-  CURRENT_CONSTRUCTOR(FullTest)(Variant<A, B, C, Empty> && q) : q(std::move(q)) {}
-};
-#endif
 
-}  // namespace
+}  // namespace SMOKE_TEST_STRUCT_NAMESPACE
```

Comment:
`SMOKE_TEST_STRUCT_NAMESPACE` is a `#define`-d symbol, a bit misleading to but it here as a comment IMHO.